piccolo/testing/model_builder.py

```diff
@@ -139,11 +139,18 @@ def _randomize_attribute(cls, column: Column) -> t.Any:
             Column class to randomize.
 
         """
+        random_value: t.Any
```

Comment:
this stops mypy complaining on the following few lines where it thinks `random_value` is a float because of the first if-body, but then i set it to a list in the following one.